### ISSUE TITLE
Update for Surface Laptop 4?

### ISSUE THREAD

**Issue by hoyty76** [MicrosoftDocs/Devices-docs#91]
The Surface Laptop 4 has been out for a few months. Can this be updated to reflect instructions for that.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6b195b1b-81bf-310e-3ff4-e520826e7e94
* Version Independent ID: fb399be5-9fb5-a3f9-f912-ed3fde235f05
* Content: [How to enable the Surface Laptop keyboard during MDT deployment - Surface](https://docs.microsoft.com/en-us/surface/enable-surface-keyboard-for-windows-pe-deployment)
* Content Source: [surface/enable-surface-keyboard-for-windows-pe-deployment.md](https://github.com/microsoftdocs/devices-docs/blob/public/surface/enable-surface-keyboard-for-windows-pe-deployment.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @Teresa-Motiv
* Microsoft Alias: **v-tea**

**Comment by bmkaiser** [comment]
Agreed. Our hardware supplier only recently got these in stock after months of delay, so I was surprised to find this article hadn't been updated already.
